File: core/src/org/rexcellentgames/burningknight/entity/item/accessory/equippable/RedBalloon.java
package org.rexcellentgames.burningknight.entity.item.accessory.equippable;

public class RedBalloon extends Equippable {
	@Override
	public void onEquip(boolean load) {
		super.onEquip(load);
		this.owner.flight += 1;
	}

	@Override
	public void onUnequip(boolean load) {
		super.onUnequip(load);
<<<<<<< HEAD
		this.owner.flight -= 1;
	}

	@Override
	public boolean canBeUpgraded() {
		return false;
=======
		this.owner.flight -=1;
>>>>>>> d5dbb7b740601360b76d810651a91d167202839d
	}
}

File: core/src/org/rexcellentgames/burningknight/entity/item/accessory/equippable/RedBalloon.kt
package org.rexcellentgames.burningknight.entity.item.accessory.equippable

class RedBalloon : Equippable() {
	override fun onEquip(load: Boolean) {
		super.onEquip(load)
		this.owner.flight += 1
	}

	override fun onUnequip(load: Boolean) {
		super.onUnequip(load)
		this.owner.flight -= 1
	}

	override fun canBeUpgraded(): Boolean {
		return false
	}

	override fun canBeDegraded(): Boolean {
		return false
	}
}